heron/common/tests/cpp/network/oserver_unittest.cpp

```diff
@@ -75,5 +75,5 @@ void OrderServer::Terminate() {
 
 void OrderServer::HandleTerminateMessage(Connection* _connection __attribute__((unused)),
                                          TerminateMessage* _message __attribute__((unused))) {
-  AddTimer([this]() { this->Terminate(); }, 1);
+  AddTimer([this]() { std::cout << "OrderServer:Terminate"; this->Terminate(); }, 1);
```

Comment:
This printout is added for future diagnosis. 
The timeout issue was not reproduced. I suspect the timeout issue may be due to this timer action. 
It does not print a lot since it is supposed to be called only once.